synapse/models/planning.py

```diff
@@ -0,0 +1,155 @@
+import synapse.lib.module as s_module
+
+class PlanModule(s_module.CoreModule):
+
+    def getModelDefs(self):
+        return (('plan', {
+            'types': (
+                ('plan:system', ('guid', {}), {
+                    'doc': 'A planning or behavioral analysis system that defines phases and procedures.'}),
+
+                ('plan:phase', ('guid', {}), {
+                    'doc': 'A phase within a planning system which may be used to group steps within a procedure.'}),
+
+                ('plan:procedure', ('guid', {}), {
+                    'doc': 'A procedure consisting of steps.'}),
+
+                ('plan:procedure:type:taxonomy', ('taxonomy', {}), {
+                    'interfaces': ('meta:taxonomy',),
+                    'doc': 'A taxonomy of procedure types.'}),
+
+                ('plan:procedure:variable', ('guid', {}), {
+                    'doc': 'A variable used by a procedure.'}),
+
+                ('plan:procedure:step', ('guid', {}), {
+                    'doc': 'A step within a procedure.'}),
+
+                ('plan:procedure:link', ('guid', {}), {
+                    'doc': 'A link between steps in a procedure.'}),
+            ),
+
+            'forms': (
+                ('plan:system', {}, (
```

Comment:
Do we want to add a :desc field?

Reply:
We have 44 properties with the relname "summary" and 97 properties with the relname "desc". We should probably lean into one of them for consistency.

Reply:
I could see the utility of adding a `:summary` field here. I'll tweak it. 👍 @vEpiphyte i think the use of `summary` has been growing for specific use cases that are a bit more abstract or self defined. I've also been veering toward property names that are a tiny bit friendlier vs saving a couple characters ( partially due to indexing pressure relief via the use of abrv translations everywhere :D )